pkg/clusteragent/admission/mutate/common.go

```diff
@@ -153,3 +155,62 @@ func containsVolumeMount(volumeMounts []corev1.VolumeMount, element corev1.Volum
 	}
 	return false
 }
+
+// isApmInstrumentationEnabled indicates if Single Step Insstrumentation is enabled for the namespace in the cluster
+// Single Step Instrumentation is enabled if:
+//   - cluster-wide configuration `apm_config.instrumentation.enabled` is true and the namespace is not excluded via `apm_config.instrumentation.disabled_namespaces`
+//   - cluster-wide configuration `apm_config.instrumentation.enabled` is false and the namespace is whitelisted via `apm_config.instrumentation.disabled_namespaces`
+func isApmInstrumentationEnabled(namespace string) bool {
```

Comment:
do we still have the need to exclude some namespaces all the time? If so, is that checked here or somewhere else?